### PR TITLE
Create message_checker.py

### PR DESCRIPTION
**Pull request by ksIsCute**
just a family friendly version of yoni's attempt

## Summary
Just another example explaining how you can use message and message.content to look for some words inside of the message sent

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [x] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/message_checker.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import voltage
+
+client = voltage.Client()
+
+@client.listen('message')
+async def on_message(message):
+  ungodly_words = ["sus", "baka", "suppose", "real", "amogus"]
+  if message.content.lower() in ungodly_words:
```
Comment (EnokiUN) [inline examples/message_checker.py]: but why

Comment (EnokiUN) [reply examples/message_checker.py]: also as I've mentioned, this is quite valueless as an example, add a command that adds words that are censored using the commands framework and I'll consider it I suppose.

File: examples/message_checker.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-@client.listen('message')
-async def on_message(message):
-  ungodly_words = ["sus", "baka", "suppose", "real", "amogus"]
-  if message.content.lower() in ungodly_words:
-    await message.reply("*GASP!* You can't say that!")
-    await message.delete()
+@client.listen('message') # specify what we're listening to
+async def on_message(message): # the name for this can be anything you want it to be
+  ungodly_words = ["sus", "baka", "suppose", "real", "amogus"] # create our list to iterate through later
+  if message.content.lower() in ungodly_words: # run the if statement to trigger if a message is in the array
```
Comment (EnokiUN) [inline examples/message_checker.py]: Check doesn't work as intended, try `if any([word in message.content.lower() for word in ungodly words])` I suppose

**Review by EnokiUN — CHANGES_REQUESTED** [review]

File: examples/message_checker.py (hunk 1 above)
Comment (ksIsCute) [reply examples/message_checker.py]: yeah epic ok

**Review by ksIsCute — COMMENTED** [review]
like dees? seems a bit complucation dont u suppose

**Comment by EnokiUN** [comment]
What do you mean? You're just altering a dictionary I suppose.

**Comment by ksIsCute** [comment]
> What do you mean? You're just altering a dictionary I suppose.

i used the if any method you gave me

**Comment by ksIsCute** [comment]
im gonna add another example inside of that file relating to on_message

**Review by ksIsCute — COMMENTED** [review]
[changes](examples/message_checker.py)
> made some changes and added the other example

**Comment by EnokiUN** [comment]
1. Why did you add another example I suppose.
2. Where's the command I requested I suppose.

**Comment by ksIsCute** [comment]
[is this was you were looking for?](https://github.com/EnokiUN/voltage/pull/37/files#diff-d197ca5e6f0d968029ae4a92b2ef1ba457d9c2f579a41284e6671ffbbfd6d8a0)

**Review by ksIsCute — COMMENTED** [review]
just wanted to know is `delete_after` added so i dont need line 14?

**Comment by ksIsCute** [comment]
```py
import voltage #import the voltage module
from voltage.ext import commands # importing commands for our client

client = commands.CommandsClient("!") # initialize our client

ungodly_words = ["sus", "baka", "suppose", "real", "amogus"] # create our list to iterate through and add items later

@client.listen('message') # specify what we're going to listen to
async def on_message(message): # the name for this can be anything you want it to be
  if any([word in message.content.lower() for word in ungodly words]): # run the if statement to trigger if the message has words in the array
    await message.reply("*GASP!* You can't say that word!", delete_after=5) # reply to the message sent and delete OUR message after 5 seconds
    await message.delete() # delete the USERS message
  await bot.handle_commands(message) 
  # Handle afterwards so other commands will work after the on_message, 

@commands.command(name="add_word") # Create our command
async def add_word(ctx, word): # Define our command
  ungodly_words.append(word.lower()) # Append to our list
  await ctx.send(f"Added `{word.lower()}` to the list of `{len(ungodly_words)}` words!") # Tell user that the word was added to the list.
  
  
client.run("TOKEN") # Run the bot
```

**Review by EnokiUN — APPROVED** [review]

**Review by RGBCube — CHANGES_REQUESTED** [review]
you should note that the added words wont be permanent, and will be discarded on every restart

**Comment by ksIsCute** [comment]
> you should note that the added words wont be permanent, and will be discarded on every restart

this doesnt make sense because then id have to impliment like a database or a json file and it will get way too overcomplicated and completely overlap the point of the example

**Comment by RGBCube** [comment]
> > you should note that the added words wont be permanent, and will be discarded on every restart
> 
> this doesnt make sense because then id have to impliment like a database or a json file and it will get way too overcomplicated and completely overlap the point of the example

no, just *say* it wont be persistent, no need to do that

**Review by RGBCube — APPROVED** [review]
LGTM

**Pull request merged**